modules/coo-troubleshooting-ui-plugin-creating-alert.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+
+// * observability/cluster_observability_operator/ui_plugins/troubleshooting-ui-plugin.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="coo-troubleshooting-ui-plugin-creating-alert_{context}"]
+= Creating the example alert
+
+
+To trigger an alert as a starting point to use in the troubleshooting UI panel, you can deploy a container that is deliberately misconfigured.
+
+.Procedure
+
+. Use the following YAML, either from the command line or in the web console, to create a broken deployment in a system namespace:
++
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: bad-deployment
+  namespace: default <1>
+spec:
+  selector:
+    matchLabels:
+      app: bad-deployment
+  template:
+    metadata:
+      labels:
+        app: bad-deployment
+    spec:
+      containers: <2>
+      - name: bad-deployment
+        image: quay.io/openshift-logging/vector:5.8
+----
+<1> The deployment must be in a system namespace (such as `default`) to cause the desired alerts.
+<2> This container deliberately tries to start a `vector` server with no configuration file. The server will log a few messages, and then exit with an error. Alternatively, you can deploy any container you like that is badly configured, causing it to trigger an alert.
```

Comment:
```suggestion
<2> This container deliberately tries to start a `vector` server with no configuration file. The server logs a few messages, and then exits with an error. Alternatively, you can deploy any container you like that is badly configured, causing it to trigger an alert.
```
See the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense).

Reply:
Done